spacemacs/errors.vim

```diff
@@ -12,3 +17,13 @@ vnoremap <leader>en    <Esc>:action GotoNextError<CR>
 let g:WhichKeyDesc_Errors_PreviousErrorAlt = "<leader>ep previous-error"
 nnoremap <leader>ep    :action GotoPreviousError<CR>
 vnoremap <leader>ep    <Esc>:action GotoPreviousError<CR>
+
+" Edit inspection settings
+let g:WhichKeyDesc_Errors_OpenInspectionSettings = "<leader>es inspection-settings"
+nnoremap <leader>es    :action PopupHector<CR>
```

Comment:
So, `:action EditInspectionSettings` is a noop now.    And when I bring it up manually by going to `Analyze -> Configure Current File Analysis... -> Configure Inspection...` ideavim is telling me it can't detect an action id for that menu.  

Maybe Jetbrains removed this action in a very recent update?  I'm running Intellij Ultimate 2020.3.2.  Do you want to check if `:action EditInspectionSettings` works for you?

Reply:
Nothing happens for me if I run that action, but I don't remember which was the previous behavior.
I don't even have the time to look into this unfortunately.
If you say that before it was working I trust you. At least with PopupHector something happens :P